docs/docsite/rst/dev_guide/developing_collections.rst

```diff
@@ -470,24 +470,69 @@ Collection versions use `Semantic Versioning <https://semver.org/>`_ for version
 
 .. _migrate_to_collection:
 
-Migrating Ansible content to a collection
-=========================================
+Migrating Ansible content to a different collection
+====================================================
+
+To migrate content from one collection to another, you need to create three PRs as follows:
+
+#. Create a PR against the old collection to remove the content.
+#. Create a PR against the new collection to add the files removed in step 1.
+#. Update the ``ansible/ansible:devel`` branch entries for all files moved.
+
+
+Removing the content from the old collection
+----------------------------------------------
+
+Create a PR against the old collection repo to remove the modules, module_utils, plugins, and docs_fragments related to this migration:
+
+#. If you are moving an action plugin, include the corresponding module with documentation.
```

Comment:
I had to read this a couple of times before I understood it. How about this (updated to match the next items' phrasing)?
```suggestion
#. If you are moving an action plugin, remove the corresponding module that contains the documentation.
```

Reply:
@gundalow @felixfontein please review this edit. I think it should be 'move' not 'remove'

Reply:
It should be removed from the old collection. This part is about removing things from the old collection, so I think it should say "remove". It should also say "removing" instead of "moving" in the beginning IMO.